system_tests/vision.py

```diff
@@ -507,3 +509,53 @@ def test_detect_properties_filename(self):
         image = client.image(filename=FACE_FILE)
         properties = image.detect_properties()
         self._assert_properties(properties)
+
+
+class TestVisionBatchProcessing(BaseVisionTestCase):
```

Comment:
I'm kind of following a pattern there. Which previous `TestCase` would you suggest putting it in?